test/spec/controllers/createPatientRequestMapper.spec.js

```diff
@@ -0,0 +1,141 @@
+'use strict';
+
+describe('CreatePatientRequestMapper', function () {
+
```

Comment:
I know there is a document started by @larslemos about test directories names, I'm not sure but I think that this test should be in: `test/spec/registration/mappers` if we are making it equal with the location of the resource being tested or `test/spec/mappers` if we are separating by duty.